src/main/java/org/gitlab/api/models/GitlabMergeRequest.java

```diff
@@ -46,23 +54,64 @@
     @JsonProperty("milestone_id")
     private Integer milestoneId;
 
-    @JsonProperty("updated_at")
-    private Date updatedAt;
+    @JsonProperty("work_in_progress")
+    private Boolean workInProgress;
 
-    @JsonProperty("created_at")
-    private Date createdAt;
+    @JsonProperty("merge_when_pipeline_succeeds")
+    private Boolean mergeWhenPipelineSucceeds;
 
-    @JsonProperty("merge_commit_sha")
-    private String mergeCommitSHA;
-    
     @JsonProperty("merge_status")
     private String mergeStatus;
 
+    @JsonProperty("sha")
+    private String sha;
+
+    @JsonProperty("merge_commit_sha")
+    private String mergeCommitSHA;
+
+    @JsonProperty("user_notes_count")
+    private Integer userNotesCount;
+
+    @JsonProperty("discussion_locked")
+    private Boolean discussionLocked;
+
+    @JsonProperty("should_remove_source_branch")
+    private Boolean shouldRemoveSourceBranch;
+
+    @JsonProperty("force_remove_source_branch")
+    private Boolean forceRemoveSourceBranch;
+
     @JsonProperty("web_url")
     private String webUrl;
 
-    @JsonProperty("sha")
-    private String sha;
+    private Boolean squash;
+
+    @JsonProperty("changes_count")
+    private Integer changesCount;
+
+    @JsonProperty("merged_by")
+    private GitlabUser mergedBy;
+
+    @JsonProperty("merged_at")
+    private Date mergedAt;
+
+    @JsonProperty("closed_by")
+    private GitlabUser closedBy;
+
+    @JsonProperty("closed_at")
+    private Date closedAt;
+
+    @JsonProperty("latest_build_started_at")
+    private Date latestBuildStartedAt;
+
+    @JsonProperty("latest_build_finished_at")
+    private Date latestBuildFinishedAt;
+
+    @JsonProperty("first_deployed_to_production_at")
+    private Date firstDeployedToProductionAt;
+
+    @JsonProperty("diff_refs")
```

Comment:
I don't see this in the API docs -- is this actually part of the public API?

Reply:
It was introduced in the API in 10.8 in this MR https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/18537 as part of this issue https://gitlab.com/gitlab-org/gitlab-ce/issues/20901

I guess they did not update the result in the docs here https://gitlab.com/gitlab-org/gitlab-ce/issues/20901 yet. I tried it with my GitLab instance and the API returns it.